asapdiscovery-ml/asapdiscovery/ml/inference.py

```diff
@@ -6,20 +6,28 @@
 import torch
 from asapdiscovery.data.openeye import oechem
 from asapdiscovery.data.postera.manifold_data_validation import TargetTags
-from asapdiscovery.ml.dataset import DockedDataset, GraphInferenceDataset
+from asapdiscovery.data.schema_v2.ligand import Ligand
+from asapdiscovery.ml.dataset import DockedDataset, GraphDataset
 from asapdiscovery.ml.models import (
     ASAPMLModelRegistry,
     LocalMLModelSpec,
     MLModelRegistry,
     MLModelSpec,
-    MLModelType,
 )
 
 # static import of models from base yaml here
 from asapdiscovery.ml.utils import build_model, load_weights
 from dgllife.utils import CanonicalAtomFeaturizer
+from mtenn.config import ModelType
 from pydantic import BaseModel, Field
 
+"""
+TODO
```

Comment:
Does this need to be done before this PR can be completed? Are all the older models invalid?

Reply:
definitely can be pushed to a separate PR. we might actually have to do some thinking about how to unify your `MLModelSpec` class with the new schema, but for now as long as the `inference` tests are passing I think we're fine